tests/functional/pv/space_reclaim/test_rbd_reclaimspace_cronjob.py

```
@@ -152,14 +152,21 @@ def test_rbd_space_reclaim_cronjob(self, pause_and_resume_cluster_load):
     @skipif_hci_provider_and_client
     @skipif_external_mode
     @pytest.mark.parametrize(
-        argnames=["replica", "compression", "volume_binding_mode", "pvc_status"],
+        argnames=[
+            "replica",
+            "compression",
+            "volume_binding_mode",
+            "pvc_status",
+            "erasure_coded",
+        ],
```

Ruff (0.15.14)

[warning] Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)